CONTRIBUTING.md

```diff
@@ -61,6 +61,7 @@ requests_).
  - (Martin Dix)
  - (Ivor Blockley)
  - Alexander Paulsell
+ - David Sutherland
```

Comment:
Bingo! :tada:

Reply:
> > `# Get the current environment (maybe os module available in jinja?)`
> 
> https://cylc.github.io/cylc/doc/built-sphinx/suite-config.html#accessing-environment-variables-with-jinja2

Ah yes, I've even used that myself 😆   ... Didn't bother looking it up again, as the issue still stands.